docs/asciidoc/modules/ROOT/examples/generated-documentation/apoc.trigger.install.adoc

```diff
@@ -0,0 +1,5 @@
+¦xref::overview/apoc.trigger/apoc.trigger.install.adoc[apoc.trigger.install icon:book[]] +
+
+`CALL apoc.trigger.install(databaseName, name, statement, selector, config) | add a trigger kernelTransaction under a name, in the kernelTransaction you can use $createdNodes, $deletedNodes etc., the selector is {phase:'before/after/rollback/afterAsync'} returns previous and new trigger information. Takes in an optional configuration.`
```

Comment:
```suggestion
`CALL apoc.trigger.install(databaseName, name, statement, selector, config) | eventually adds a trigger for a given database which is invoked when successful write transactions occur.
```

Reply:
A slight problem I see, is that in the signature we're using an input parameter called "kernelTransaction" whereas in the docs we are interchangeably using either "kernelTransaction" or "statement". If possible, then I think some consistency would be nice. 

PS. I would probably prefer we use "statement" everywhere as "kernelTransaction" is a not really what this input parameter represents. Essentially the input is a "statement" which will be run after each successful "write transaction".

Reply:
This problem also occurs in the apoc.trigger.install procedure page.

Reply:
Changed